src/main/java/com/conveyal/analysis/models/AggregationArea.java

```diff
@@ -1,14 +1,33 @@
 package com.conveyal.analysis.models;
 
+import com.conveyal.analysis.UserPermissions;
 import com.fasterxml.jackson.annotation.JsonIgnore;
 
 /**
  * An aggregation area defines a set of origin points to be averaged together to produce an aggregate accessibility figure.
  * It is defined by a geometry that is rasterized and stored as a grid, with pixels with values between 0 and 100,000
  * depending on how much of that pixel is overlapped by the mask.
  */
-public class AggregationArea extends Model {
+public class AggregationArea extends BaseModel {
```

Comment:
This has all been clarified - when polygons are not fused we get many aggregation areas from a single spatial data source. These are then grouped into DataGroups, because we might produce one group per zoom level for example, all from the same source. I updated the javadoc as well.